tools/pd-ctl/pdctl/command/scheduler.go

```diff
@@ -442,10 +442,31 @@ func NewConfigSchedulerCommand() *cobra.Command {
 		newConfigHotRegionCommand(),
 		newConfigShuffleRegionCommand(),
 		newConfigGrantHotRegionCommand(),
+		newConfigBalanceLeaderCommand(),
```

Comment:
How about other scheduler.

Reply:
> How about other scheduler.

other schedulers should modify their config handler firstly